flake.nix

```diff
@@ -142,7 +142,6 @@
                 types_or = [ "markdown" ];
                 pass_filenames = true;
               };
-              nixpkgs-fmt.enable = true;
```

Comment:
@alxiong this is an older version of the file. I guess your `main` was out of date?

Reply:
sorry about that, fixed